library/src/main/java/com/novoda/downloadmanager/lib/NotificationCustomiser.java

```diff
@@ -2,7 +2,9 @@
 
 import android.support.v4.app.NotificationCompat;
 
+import com.novoda.downloadmanager.Download;
+
 public interface NotificationCustomiser {
 
-    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, long batchId);
+    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, Download download);
```

Comment:
yeah well this is the DLM library so in order to make it simpler for clients the idea was to leak less impl. details I think. you can discuss more here if you want https://github.com/novoda/download-manager/issues/85 :smiley: